sikhifm_db_back.sql

```diff
@@ -0,0 +1,265 @@
+-- -------------------------------------------------------------
+-- TablePlus 3.6.2(322)
+--
+-- https://tableplus.com/
+--
+-- Database: SikhiFM
+-- Generation Time: 2020-07-06 11:14:21.9310
+-- -------------------------------------------------------------
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+CREATE DATABASE `sikhifm_db`;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`Album`;
+CREATE TABLE `sikhifm_db`.`Album` (
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
@sufikaur @tsingh777 @ManjotS is there a reason why we are using both ID and sID?

Reply:
we used sid on previous sites as an obfuscator. Basically if we say load track 11, a spider can come through and increment to 12, 13, 14 etc..

If we have it U8G5FC its not a predictable pattern. 

For performance reasons though, we don't use this as the PK. 

We could do a trick that increments random looking letters to meet both use cases after internship.

Reply:
will the primary key be something a spider will ever be able to see?

Reply:
we can iterate on this